src/shared/calendar/calendar.tsx

```diff
@@ -0,0 +1,43 @@
+import DayOfWeek from './day-of-week'
+import CalendarNavigator from './calendar-navigator'
+import CalendarBody from './calendar-body'
+import useCalendar from '../hooks/useCalendar'
+
+export default function Calendar() {
+  const {
+    selectedDate,
+    currentMonthAllDates,
+    handleNextMonth,
+    handlePrevMonth,
+    handleNextYear,
+    handlePrevYear,
+    handleselectedDate,
+    year,
+    month,
+  } = useCalendar()
+
+  return (
+    <>
+      <CalendarNavigator
+        next={'다음년'}
+        prev={'이전년'}
+        text={year + '년'}
+        handlePrev={handlePrevYear}
+        handleNext={handleNextYear}
+      />
+      <CalendarNavigator
+        next={'다음달'}
+        prev={'이전달'}
+        text={month + '월'}
```

Comment:
P2

```suggestion
      <CalendarNavigator
        next={'다음달'}
        prev={'이전달'}
        month={currentMonth}
```

CalendarNavigator는 결국 캘린더 조작을 할 수 있는 컴포넌트인데 문자열로 조합을 해주는게 좋을까요? 저라면 month만 넘길것 같아요. "월"은 무조건 들어가게 되어있는데 이걸 사용하는 측에서 작성한다는건 불편할 것 같습니다